src/katex.less

```diff
@@ -4,6 +4,8 @@
 // The mu unit is defined as 1/18 em
 @mu: 1/18em;
 
+@version: "";
```

Comment:
Is there something that replaces the `""` with the actual version string from package.json?

Reply:
@kevinbarabash 
https://github.com/KaTeX/KaTeX/blob/db184677ea34b49ebbe0b9266816b7431291c5cd/webpack.common.js#L64-L66
sets the version.

Reply:
Could you add a comment here before merging so people don't accidental "fix" this?